### PR TITLE
Remove Code Studio references from privacy policy

### PR DESCRIPTION
**Pull request by poorvasingal**
Related PR: https://github.com/code-dot-org/code-dot-org/pull/15762

### CODE REVIEW

File: dashboard/app/views/home/_privacy.md [hunk 1]
```diff
@@ -23,49 +22,49 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
...
 
-Our Privacy Policy describes what information we gather from our Users, how we use that information, and what we do to protect it. Your use of the Website and any information you provide on the Website are subject to the terms of this Privacy Policy, which is incorporated into and subject to our Terms of Service. Any capitalized terms used but not defined within this Privacy Policy have the meaning given to them in our [Terms of Service](https://code.org/tos). By visiting or using our Website, you expressly consent to the information handling practices described in this Privacy Policy.
+Our Privacy Policy describes what information we gather from our Users, how we use that information, and what we do to protect it. Your use of the Website and any information you provide on the Website are subject to the terms of this Privacy Policy, which is incorporated into and subject to our Terms of Service. Any capitalized terms used but not defined within this Privacy Policy have the meaning given to them in our [Terms of Service](/tos). By visiting or using our Website, you expressly consent to the information handling practices described in this Privacy Policy.
```
Comment (tanyaparker) [inline dashboard/app/views/home/_privacy.md]: This is on dashboard so the link has to stay `[Terms of Service](https://code.org/tos)`. There is no studio.code.org/tos.

File: dashboard/app/views/home/_privacy.md [hunk 2]
```diff
@@ -23,49 +22,49 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
...
-Login time, IP address, and other technical data | Mandatory (automatic) | This data helps Code.org troubleshoot any problems users experience. It also helps Code.org understand usage patterns, to guide the build out of servers to support all users, and to enable updating the site while minimizing service disruption. See section below on ["technical information"](https://code.org/privacy/#techinfo) for additional details.
+Login time, IP address, and other technical data | Mandatory (automatic) | This data helps Code.org troubleshoot any problems users experience. It also helps Code.org understand usage patterns, to guide the build out of servers to support all users, and to enable updating the site while minimizing service disruption. See section below on [“technical information”](#techinfo) for additional details.
```
Comment (tanyaparker) [inline dashboard/app/views/home/_privacy.md]: Same here `(https://code.org/privacy/#techinfo)`

File: dashboard/app/views/home/_privacy.md [hunk 3]
```diff
@@ -23,49 +22,49 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
...
+Progress in the course <br>1- Date/Time each stage is tried<br>2- Number of tries to solve a puzzle, and whether it was solved successfully or optimally<br>3- Information on how the student solved the puzzle including time to completion and whether they used hints<br>4- The code that the student submitted<br>5- Student-provided answers to simple assessments (e.g. multiple-choice questions) | Mandatory (automatic) | This information is displayed to students and their teachers to see their progress in a course, to see the code they’ve created, and to identify topics they need help with. It also lets students pick up where they left off if they log out and login later. See [example progress report](http://code.org/files/student-progress.pdf), and see section below on [“technical information”](#techinfo) for additional details.<br><br>This data also helps Code.org improve the course effectiveness. For example, if a puzzle is too hard, Code.org may take action (like [providing better hints](http://code.org/hints)) to improve the learning process.
```
Comment (tanyaparker) [inline dashboard/app/views/home/_privacy.md]: And here `["technical information"](https://code.org/privacy/#techinfo)`

File: dashboard/app/views/home/_privacy.md [hunk 4]
```diff
@@ -23,49 +22,49 @@ Our Privacy Policy is fueled by our commitment to the following Privacy Principl
...
+Data collected by student-created apps | Optional | Students may use Code.org to create their own apps. Depending on the app author’s design, a student-created app may in turn collect data by prompting other Users (anybody who tries using the student-created app) to enter information, such as a favorite movie.<br><br>If a student creates an app that collects and stores data in this fashion, all data entered by users of the app may be accessed and possibly shared publicly by the app author,  the app itself, and potentially anybody with a link to view the app. Code.org does not itself use or share this data outside of the app.<br><br>Before using a student-created app that collects data, Users are shown a clear warning that any data they enter may be  shared publicly and that they should not share anything personal to them or to others.
```
Comment (tanyaparker) [inline dashboard/app/views/home/_privacy.md]: Double space `shared publicly by the app author,  the app itself`

File: dashboard/app/views/home/_privacy.md [hunk 5]
```diff
@@ -121,13 +120,13 @@ Certain capabilities linked from the Code.org Website may be implemented by thir
...
 
-Code.org has created a tool called the [“Internet Simulator”](https://studio.code.org/s/netsim) for use in High School classrooms to model how the Internet functions. With this tool, students participating in a teacher-supervised classroom activity can send text-based messages to their teacher and to other students in their specific classroom section. Message contents are visible to the classroom teacher and are not accessed or used by Code.org for any purpose other than in this educational tool.  All messages are deleted after two hours of class inactivity, or upon a manual reset by the teacher.
+Code.org has created a tool called the [“Internet Simulator”](/internetsimulator) for use in High School classrooms to model how the Internet functions. With this tool, students participating in a teacher-supervised classroom activity can send text-based messages to their teacher and to other students in their specific classroom section. Message contents are visible to the classroom teacher and are not accessed or used by Code.org for any purpose other than in this educational tool.  All messages are deleted after two hours of class inactivity, or upon a manual reset by the teacher.
```
Comment (tanyaparker) [inline dashboard/app/views/home/_privacy.md]: Same `[“Internet Simulator”](https://studio.code.org/s/netsim)`

File: dashboard/app/views/home/_privacy.md [hunk 6]
```diff
@@ -144,20 +143,20 @@ Code.org does not directly collect or store payment instruments. Donations to Co
...
+If Code.org learns that it has inadvertently collected Personal Information or Persistent Identifiers from children under the age of 13 without prior parental or teacher consent, Code.org will take appropriate steps to delete this information. If you are a teacher, parent or legal guardian of a student on Code.org, you can ask us to deactivate the student’s account, delete any hashed email address or inadvertently collected Personal Information or Persistent Identifiers, and request that we no longer allow the student to submit their information to Code.org. To make such a request, please contact us at [http://code.org/contact](http://code.org/contact). Before processing your request, we may verify your identity and your relationship with the student.
```
Comment (tanyaparker) [inline dashboard/app/views/home/_privacy.md]: Should keep as `https` no?

File: dashboard/app/views/home/_privacy.md (hunk 3 above)
Comment (tanyaparker) [reply dashboard/app/views/home/_privacy.md]: Also do we want to update these two code.org links to `https` both in this file and pegasus file?

**Comment by poorvasingal** [comment]
Sorry @tanyaparker - I was trying to copy / paste from the other one since there were so many changes. Ugh, I hate having 2 copies. Thank you SO much for looking over my stuff. Does it look okay now?

**Comment by tanyaparker** [comment]
@breville not that this is work to do now, but is it possible to combine the tos and privacy policy into 1 thing between pegasus and dashboard? Or does that only work for haml views and not markdown?

**Comment by breville** [comment]
Yes, this should be possible, and it looks like we can probably even do it server-side using .md files, the same way we are sharing .haml server-side now.

**Comment by tanyaparker** [comment]
Cool! I'll track as maybe tech debt to do in August/September.

**Pull request merged**